packages/start/src/server-handler/index.tsx

```diff
@@ -23,28 +26,44 @@ async function handleServerAction(event: H3Event) {
 export async function handleServerRequest(request: Request, _event?: H3Event) {
   const method = request.method
   const url = new URL(request.url, 'http://localhost:3000')
+  // extract the serverFnId from the url as host/_server/:serverFnId
+  // Define a regex to match the path and extract the :thing part
+  const regex = /\/_server\/([^/?#]+)/
```

Comment:
This base of `"_server"` needs to take in the inject environment variable of `"TSS_SERVER_BASE"` like its being done in `packages/start/src/client-runtime/getBaseUrl.tsx`.